### DISCUSSION TITLE
Direct Binding for Enbrighten Switches

### DISCUSSION THREAD

**Discussion by deviantintegral** [Koenkk/zigbee2mqtt#18768] [Devices]
I've wanted to set up direct binding between [two Enbrighten switches](https://www.zigbee2mqtt.io/devices/43076.html#enbrighten-43076) (model 43076) for a while and wasn't able to get it working. As well, I'd previously wanted to be able to trigger an automation in Home Assistant even if the light was already on, to extend out a timer.

I came across https://old.reddit.com/r/homeassistant/comments/seyywm/zigbee2mqtt_enbrighten_switches_dont_report/ which gave me a hint! The trick is to use `2` as the source endpoint instead of one. Unlike that post, I'm unable to bind to a destination on 2, so instead it's `source endpoint: 2`, `destination: <the device>`, `destination endpoint: 1`, `clusters: OnOff`, along with a second bind with the destination set to the coordinator. Without the coordinator bind, the bind to the other switch doesn't trigger.

Here's the full binding configuration. I imagine this works with their dimmers and other switches too.

<img width="1722" alt="image" src="https://github.com/Koenkk/zigbee2mqtt/assets/255023/3392144f-d9f9-427a-a188-1ab7d0ca64ac">

**Comment by matt-sorenson** [comment]
@deviantintegral were you ever able to figure this out? I'm running into the same issue

**Reply by deviantintegral** [reply]
I just took a look, and this is now working for me even without the direct coordinator bind on endpoint 2:

<img width="1771" height="582" alt="image" src="https://github.com/user-attachments/assets/d4ac727d-3a5a-485c-b77c-adfaf06c1444" />

**Reply by matt-sorenson** [reply]
Thanks for looking so quickly after so long. I copied your setup and it worked!
